source/includes/_architecture_deep_linking.md

```diff
@@ -0,0 +1,76 @@
+# Architecture Deep Linking
+
+## Content
+
+The following information explains how to generate a deep link to take a user to view all of their content under a
+specific architecture.
+
+### Base URL
+
+`/#/architecture/<ARCHITECTURE_ID>/content`
+
+### Query Parameters
+
+> Content Filter Example - adding an automatic filter on the status attribute where the value contains "Sold"
+
+```
+contentFilter=%7B%22status%22%3A%20%7B%22contains%22%3A%20%22Sold%22%7D%7D
+```
+
+> The raw value from the example above is
+
+```
+{
+  "status": {
+    "contains": "Sold"
+  }
+}
+```
+
+All query parameters must be properly encoded URI components. For further information on this, please see documentation
+[here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)
+
+| Query Param    | Required | type         | Description                                          |
+| -------------- | -------- | ------------ | ---------------------------------------------------- |
+| contentFilter  | false    | Encoded JSON | JSON enabling filtering of content                   |
```

Comment:
The PDF with this documentation didn't really have any more detail then this and the example. Should we expand on this more?